assignments/callbacks.js

```diff
@@ -21,22 +21,42 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+// getLength passes the length of the array into the callback.
+ 
+function getLength(arr, cb) {  return cb(arr.length);
 }
+getLength(items, function(length) {
+ console.log(length);
+});
+
 
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
+  last(items, log);
+
+  // last passes the last item of the array into the callback.
+
 
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+// sumNums adds two numbers (x, y) and passes the result to the callback.
+function sumNums(x, y, cb){
+  return cb(x+y);
 }
+sumNums(2, 4, log)
+
+}
+
+return cb(x+y);
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
+function multiplyNums(x, y, cb) {
+  return cb(x*y);
+ }
+   multiplyNums(4,4, log)
```

Comment:
these are all the same, using log I think twisted you up. 
https://codeburst.io/javascript-what-the-heck-is-a-callback-aba4da2deced
https://www.geeksforgeeks.org/javascript-callbacks/